pkgs/development/python-modules/spacy-curated-transformers/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  curated-tokenizers,
+  curated-transformers,
+  spacy,
+  torch,
+}:
+
+buildPythonPackage rec {
+  name = "spacy-curated-transformers";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "explosion";
+    repo = name;
+    rev = "release-v${version}";
+    hash = "sha256-3LL0ofVsyacMzLJtttg0Tl9SlkPex7TwWL/HVF4WkfI=";
+  };
+
+  dependencies = [
+    curated-tokenizers
+    curated-transformers
+    spacy
```

Comment:
```suggestion
```
https://github.com/explosion/spacy-curated-transformers/blob/b6b6bd621077a38d2baa5e855ea2db47d0d2375b/setup.cfg#L16-L19

Reply:
See above, `setup.cfg` is wrong.